mailpoet/assets/js/src/segments/dynamic/store/controls.ts

```diff
@@ -31,7 +31,7 @@ export async function LOAD_SEGMENT({ segmentId }: { segmentId: number }): Promis
     });
     return {
       success: true,
-      res: convertSavedData(res.data),
+      res: convertSavedData(<{ [key: string]: string | number; }> res.data),
```

Comment:
You can also use `Record<string, string | number>`.

Reply:
Addressed in 2f0a9e80f